env/environment.go

```diff
@@ -12,6 +12,27 @@ type Environment struct {
 	env map[string]string
 }
 
+type Pair struct {
+	Old string
+	New string
+}
```

Comment:
Minor point, but I'd suggest a different name for this type.

In the spatial context of being near `type Diff` it makes sense, but in the `package env` context, I think I'd expect `env.Pair` to be `{Key string, Value string}` or something.

Maybe it should be `DiffPair` or `StringDelta` or `StringChange` or something?

Reply:
Changed in https://github.com/buildkite/agent/pull/1488/commits/b4038de920e11cf17b1329b1b2bf6414a6bb22c6